src/viam/utils.py

```diff
@@ -24,7 +24,11 @@
     from typing_extensions import ParamSpec
 
 
-def primitive_to_value(v: Any) -> Value:
+# Types that can be encoded into a protobuf `Value`
+ValueTypes = Union[bool, SupportsBytes, SupportsFloat, List, Mapping, str, None]
+
```

Comment:
Sorry, should've specified better. This should be in docstring format where the comment is enclosed by triple quotes and is below the definition:

```python3
ValueTypes = Union[bool, SupportsBytes, SupportsFloat, List, Mapping, str, None]
"""Types that can be encoded into a protobuf ``Value``"""
```